translations/README.te.md

```diff
@@ -81,4 +85,3 @@ cd React-GitHub-Resume
 ## 📄 లైసెన్స్
 
 ఈ ప్రొజెక్ట్ యం.ఐ.టి లైసెన్స్ కింద ఉన్నది. మరిన్ని విషయాల కోసం [లైసెన్స్](./LICENSE) ఫైలు చూడండి.
```

Comment:
Please, correct the `LICENSE` route to `../LICENSE`

Reply:
fixed